scriptmodules/libretrocores/lr-beetle-pce.sh

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env bash
+
+# This file is part of The RetroPie Project
+#
+# The RetroPie Project is the legal property of its developers, whose names are
+# too numerous to list here. Please refer to the COPYRIGHT.md file distributed with this source.
+#
+# See the LICENSE.md file at the top-level directory of this distribution and
+# at https://raw.githubusercontent.com/RetroPie/RetroPie-Setup/master/LICENSE.md
+#
+
+rp_module_id="lr-beetle-pce"
+rp_module_desc="PC Engine/CD/SuperGrafx emulator - Mednafen PCE port for libretro"
+rp_module_help="ROM Extensions: .7z .ccd .chd .cue .pce .sgx\n\nCopy your PC Engine roms to $romdir/pcengine\n\nCopy the required BIOS file syscard3.pce to $biosdir"
+rp_module_licence="GPL2 https://raw.githubusercontent.com/libretro/beetle-pce-libretro/master/COPYING"
+rp_module_repo="git https://github.com/libretro/beetle-pce-libretro.git master"
+rp_module_section="main"
+rp_module_flags="!armv6"
+
+function _update_hook_lr-beetle-pce() {
+    # move from old location and update emulators.cfg
+    renameModule "lr-mednafen-pce" "lr-beetle-pce"
```

Comment:
I assume this module is based on copy and paste from the pce-fast module as there is no need for this. We don't have a lr-mednafen-pce module that is being renamed.

Reply:
Understood, removed.